std/variant.d

```diff
@@ -690,12 +690,29 @@ public:
                 }
                 else static if (is(T == U[n], U, size_t n))
                 {
-                    auto p = cast(T*)(new U[n]).ptr;
-                    *p = rhs;
+                    static if (!isMutable!T)
+                    {
+                        alias UT = Unqual!T;
+                        auto p = cast(UT*)(new U[n]).ptr;
+                        *p = cast(UT) rhs;
+                    }
+                    else
+                    {
+                        auto p = cast(T*)(new U[n]).ptr;
```

Comment:
You can just drop this branch and use `Unqual` everywhere since `Unqual!S == S`

Reply:
Thanks!
It has been handled as review required.